src/openrpc/media_info.json

```diff
@@ -0,0 +1,425 @@
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": [
+                            {
+                                "codec": "hevc",
+                                "codecLevel": "4.2",
+                                "codecProfile": "main",
+                                "container": "video/mp4",
+                                "frameRate": 30,
+                                "hdr": [
+                                    "hdr10"
+                                ],
+                                "pipelineId": 1,
+                                "resolution": [
+                                    1920,
+                                    1080
+                                ]
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "audioFormat",
+            "summary": "Get the audio format currently used by the specified media pipeline.",
+            "tags": [
+                {
+                    "name": "property:readonly",
+                    "x-session": "sessionId",
+                    "x-session-name": "mediaSession"
+                },
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:media-info:audio-format"
+                    ]
+                }
+            ],
+            "params": [
+                {
+                    "name": "pipelineId",
+                    "required": true,
+                    "schema": {
+                        "$ref": "#/components/schemas/PipelineId"
+                    }
+                }
+            ],
+            "result": {
+                "name": "audioFormat",
+                "summary": "Audio format used by the pipeline.",
+                "schema": {
+                    "$ref": "#/components/schemas/AudioFormat"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [
+                        {
+                            "name": "pipelineId",
+                            "value": 1
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": {
+                            "bitRate": 128,
+                            "channels": "2",
+                            "codec": "aac",
+                            "container": "audio/mp4",
+                            "pipelineId": 1,
+                            "sampleRate": 48
+                        }
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onActiveAudioFormatsChanged",
```

Comment:
We don't need this method explicitly defined because activeAudioFormats has been tagged with _**property**_